cmd/ctr/app/main.go

```diff
@@ -103,6 +104,7 @@ containerd CLI
 		run.Command,
 		snapshots.Command,
 		tasks.Command,
+		install.Command,
```

Comment:
How about putting this under `ctr packages` so that we can have `ctr packages list/uninstall/update/upgrade`